source/arm9/peripherals/camera.twl.c

```diff
@@ -13,6 +13,49 @@
 
 extern u8 cameraActiveDevice;
 
+struct camera_state {
+    int8_t last_mode;
+    uint8_t  dma_scanlines; //future use
+    uint32_t dma_wlength; //
+    uint32_t dma_blength; //might be doable as u16, but better safe than sorry
+    uint16_t height;//
+    uint16_t width;//
+} __camera_state;
```

Comment:
Same ofr this - if it's not used outside of `camera.twl.c`, it is `static`.

Reply:
Also, names that start with two underscores are reserved (and names with one underscore followed by an uppercase letter).

Basically, this struct is only seen inside of this C file, so the name of the struct can be anything as long as it's `static`.